analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -228,32 +293,18 @@ histologies_df <- readr::read_tsv(opt$metadata)
 
 #### Annotation file -----------------------------------------------------------
 
-# this is the output of GenomicFeatures::makeTxDbFromGFF
-# TODO: possibly update this when the GTF file gets included in the data
-#       download; may also remove the --gtf_file option and hardcode it?
-annotation_directory <- file.path(root_dir,
-                                  "analyses",
-                                  "focal-cn-file-preparation",
-                                  "annotation_files")
-annotation_file <- file.path(annotation_directory,
-                             "txdb_from_gencode.v27.gtf.db")
-
-if (!file.exists(annotation_file)) {
-  # Define the annotations for the hg38 genome
-  txdb <- GenomicFeatures::makeTxDbFromGFF(
-    file = opt$gtf_file,
-    format = "gtf"
-  )
-  # can do this even if the directory exists
-  dir.create(annotation_directory, showWarnings = FALSE)
-  # write this to file to save time next time
-  AnnotationDbi::saveDb(txdb, annotation_file)
-} else {
-  txdb <- AnnotationDbi::loadDb(annotation_file)
-}
+# Read in prepared cds file
+cds_df <- data.table::fread(opt$cds_file, data.table = FALSE)
 
-# extract the exons but include ensembl gene identifiers
-tx_exons <- GenomicFeatures::exons(txdb, columns = "gene_id")
+# Create the GRanges object
```

Comment:
I guess its a little odd we do the same thing (making a BED-like file into a GenomicRanges object) twice (here and lines 261-266) but do it a different way. May be better to choose one approach and stick to it. The way you did it above is probably a little cleaner.